exa/__init__.py

```diff
@@ -95,7 +108,7 @@ def db_conn(self):
             EXA_DB_CONN='sqlite://'
             EXA_DB_CONN='postgresql://username:password@localhost:5432/dbname'
         """
-        return os.environ.get('EXA_DB_CONN', '')
+        return os.getenv('EXA_DB_CONN', '')
```

Comment:
I like this change. A question I would have is would it be better to be passing around the default `None` rather than an empty connection string when the env variable isn't set? In my (limited) experience, it is often better to check `is None` than to do string checks.

Reply:
Yeah the original thought was that downstream string methods won't fail and can still do appropriate checks but there is no reason not to check is None first.